### PR TITLE
test: cover consumer-gov in E2E democracy tests

### PR DESCRIPTION
**Pull request by sainoe**
## Description
The consumer gov proposals in the E2E democracy tests are currently performing a no-op as they attempt to enable a param, i.e. `transfer.params.send_enabled`, [which is already enabled per default](https://ibc.cosmos.network/v7.3.x/apps/transfer/params).

This PR fixes these tests by disabling the `send_enabled` param in the democracy tests config and asserting that the consumer gov proposal process enables it as expected. 

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

* [ ] Included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] Added `!` to the type prefix if state-machine breaking change (i.e., requires coordinated upgrade)
* [ ] Confirmed this PR does not introduce changes requiring state migrations, OR migration code has been added to consumer and/or provider modules
* [ ] Targeted the correct branch (see [PR Targeting](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#pr-targeting))
* [ ] Provided a link to the relevant issue or specification
* [ ] Followed the guidelines for [building SDK modules](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules)
* [ ] Included the necessary unit and integration [tests](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#testing)
* [ ] Added a changelog entry to `CHANGELOG.md`
* [ ] Included comments for [documenting Go code](https://blog.golang.org/godoc)
* [ ] Updated the relevant documentation or specification
* [ ] Reviewed "Files changed" and left comments if necessary 
* [ ] Confirmed all CI checks have passed
* [ ] If this PR is library API breaking, bump the go.mod version string of the repo, and follow through on a new major release for both the consumer and provider

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

* [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ] confirmed `!` in the type prefix if API or client breaking change
* [ ] confirmed this PR does not introduce changes requiring state migrations, OR confirmed migration code has been added to consumer and/or provider modules
* [ ] confirmed all author checklist items have been addressed
* [ ] reviewed state machine logic
* [ ] reviewed API design and naming
* [ ] reviewed documentation is accurate
* [ ] reviewed tests and test coverage

### CODE REVIEW

File: tests/e2e/config.go [hunk 1]
```diff
@@ -243,11 +243,12 @@ func DefaultTestConfig() TestConfig {
 
 func DemocracyTestConfig(allowReward bool) TestConfig {
 	consumerGenChanges := ".app_state.ccvconsumer.params.blocks_per_distribution_transmission = \"20\" | " +
-		".app_state.gov.voting_params.voting_period = \"10s\" | " +
+		".app_state.gov.params.voting_period = \"10s\" | " +
```
Comment (sainoe) [inline tests/e2e/config.go:246]: Gov props weren't executed due to this typo.

Comment (p-offtermatt) [reply tests/e2e/config.go:246]: Seems bad that small things like this break things (particularly that this seems to break them silently..), good catch!

File: tests/e2e/main.go [hunk 2]
```diff
@@ -173,8 +173,12 @@ func getTestFileUsageString() string {
 
 	// Test runner selection
 	builder.WriteString("Test runner selection:\nSelection of test runners to be executed:\n")
+	testConfigSet := map[string]struct{}{}
```
Comment (sainoe) [inline tests/e2e/main.go:176]: Small change to remove duplicates in the test runner selection.

**Review by shaspitz — APPROVED** [review]
Nice catch!

**Pull request merged**